pyServer/manifests/linux/monitor-mgmt

```diff
@@ -1,39 +1,49 @@
-echo,#Microsoft Monitoring Agent for Linux Log File
```

Comment:
Microsoft Monitoring Agent is only used to refer to the Log Analytics Windows Agent